React/tic-tac/src/redux/auth/reducer.js

```diff
@@ -0,0 +1,13 @@
+function reducerAuth(state = false, action) {
+  switch (action.type) {
+    case 'LogIn':
```

Comment:
If you create the object with the action types, you can call those types here. 😺